examples/proxy/record-encryption/PREPARE_KMS.md

```diff
@@ -127,3 +129,67 @@ Finally, delete the .token files:
 ```sh
 rm *.token
 ```
+
+# Preparing AWS KMS
+
+To use [AWS KMS](https://docs.aws.amazon.com/kms/latest/developerguide/overview.html) for record encryption, follow these steps. 
+
+### Prerequisites
+
+* An AWS account with the following privileges:
+** The ability to create keys in AWS KMS.
```

Comment:
if the intention here is to create a nested bullet, shouldn't be used with a single asterisk indented? [nested lists](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#nested-lists)

```
* An AWS account with the following privileges:
  * The ability to create keys in AWS KMS.
```

Reply:
Yes, you are quite correct.